.github/workflows/test_suite.yaml

```diff
@@ -20,86 +20,85 @@ jobs:
       ENV_NAME: tests
       PYTHON: ${{ matrix.python-version }}
       OS: ${{ matrix.os }}
+      CC: gcc
     name: Testing
-    #    needs: [linter]
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: [3.6, 3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9, "3.10"]
+    defaults:
+     run:
+       # Adding -l {0} ensures conda can be found properly in each step
+       shell: bash -l {0}
     steps:
       - uses: actions/checkout@master
         with:
           fetch-depth: 0
-      - name: Get Miniconda MacOSX
```

Comment:
so nowadays, there is no need to distinguish macos and ubuntu?

Reply:
That's correct, it has its own action that will determine that.